### PR TITLE
pkp/pkp-lib#9950 Add laravel pagination support for useFetchPaginated

### PR DESCRIPTION
**Pull request by jardakotesovec**

### CODE REVIEW

File: src/utils/i18n.js [hunk 1]
```diff
@@ -16,7 +16,7 @@
 export function t(key, params) {
 	if (
 		typeof pkp === 'undefined' ||
-		typeof pkp.localeKeys[key] === 'undefined'
+		typeof pkp?.localeKeys?.[key] === 'undefined'
```
Comment (ewhanson) [inline src/utils/i18n.js:19]: Was this and the other optional chaining additions causing issues previously, or was there something specific to these changes that drew your attention to this?

File: src/composables/useFetch.test.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ import {setActivePinia, createPinia} from 'pinia';
 
 import {useFetch} from './useFetch';
 import {useModalStore} from '@/stores/modalStore';
+global.pkp = global.pkp || {};
```
[the remark this answers is not in the document]
Comment (jardakotesovec) [reply src/composables/useFetch.test.js:18]: @ewhanson yeah, we have not had ui-library unit tests running on CI - so it regressed over time..

We are adding the it here https://github.com/pkp/pkp-lib/issues/9951

File: src/utils/i18n.js (hunk 1 above)
Comment (jardakotesovec) [reply src/utils/i18n.js:19]: It was causing issues previously.. but noticed it while working on this, when I run tests after some time.

File: src/composables/useFetch.test.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (ewhanson) [reply src/composables/useFetch.test.js:18]: Awesome, that's great!

File: src/utils/i18n.js (hunk 1 above)
Comment (ewhanson) [reply src/utils/i18n.js:19]: Figured it was something like this, but wanted to make sure I wasn't missing anything else. Thanks!

**Pull request merged**